apis/r/DESCRIPTION

```diff
@@ -6,7 +6,7 @@ Description: Interface for working with 'TileDB'-based Stack of Matrices,
     like those commonly used for single cell data analysis. It is documented at
     <https://github.com/single-cell-data>; a formal specification available is at
     <https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md>.
-Version: 1.6.99.2
+Version: 1.7.99
```

Comment:
1.6.99.3 ?

It cannot be 1.7.99 for 1.7.0 as 1.7.99 > 1.7.0 in the version-comparison methods.

Reply:
Oh penny slowly dropping.  You are _skipping ahead_ to make room for 1.7.0 ?

Reply:
@eddelbuettel I'm confused.

* When latest release was 1.6.*, `main` had 1.6.99
* Today when latest release becomes 1.7.0, I would think `main` should have 1.7.99

@mojaveazure @aaronwolen please  help me understand.

Reply:
Yes we are (by now) on the same page as I wrote five mins ago after the penny dropped.

Reply:
@eddelbuettel you & my last two posts crossed on the wire.

Yes: per our [established procedure](https://github.com/single-cell-data/TileDB-SOMA/wiki/Branches-and-releases) this is what I'm doing.

Reply:
And you may notice that I also approved the PR.

Reply:
> crossed on the wire.

It happens. All good now!

Reply:
Maybe next time, for the dense ones over here in Illinois, add three words of the 'increment ahead of the upcoming 1.7.0 release' as I was too rear-view mirror focussed at first.